client/python/datajunction/client.py

```diff
@@ -69,12 +69,17 @@ def common_dimensions(
     def common_metrics(
         self,
         dimensions: List[str],
-    ):
+    ) -> List[str]:  # pragma: no cover # Tested in integration tests
         """
         Return common metrics for a set of dimensions.
         """
-        # TODO  # pylint: disable=W0511
-        # Add this to the server APIs first.
+        query_params = [("node_type", models.NodeType.METRIC.value)]
+        for dim in dimensions:
+            query_params.append(("dimension", dim))
+        response = self._session.get(
+            f"/dimensions/common/?{urlencode(query_params)}",
```

Comment:
if you want you can actually just do this here:
```
self._session.get("/dimensions/common/", params=query_params)
```

Reply:
Sorry @shangyian I don't understand your comment.

Reply:
@agorajek I just meant that you could replace 
```
response = self._session.get(f"/dimensions/common/?{urlencode(query_params)}")
```
with
```
response = self._session.get("/dimensions/common/", params=query_params)
```
Since using the `params` arg that's provided by the requests library is just another way of doing this vs `urlencode`. But totally optional!

Reply:
Thanks @shangyian! I like the `params` arg better, but I'll adjust it next time.